x/ccv/consumer/types/genesis_test.go

```diff
@@ -16,8 +16,7 @@ import (
 	tmtypes "github.com/cometbft/cometbft/types"
 
 	"github.com/cosmos/interchain-security/v3/testutil/crypto"
-	"github.com/cosmos/interchain-security/v3/x/ccv/consumer/types"
-	ccv "github.com/cosmos/interchain-security/v3/x/ccv/types"
+	types "github.com/cosmos/interchain-security/v3/x/ccv/types"
```

Comment:
Related to the [comment](https://github.com/cosmos/interchain-security/pull/1071/files#r1300611106). I guess in these cases we can use `ccvtypes` even in this PR since the name is already changed?